tests/_support/AcceptanceTester.php

```diff
@@ -235,4 +235,19 @@ public function doAdministratorLogout()
 		$I->waitForText('Log in', 60, ['xpath' => "//fieldset[@class='loginform']//button"]);
 
 	}
+
+	/**
+	 * Selects an option in a Joomla Radio Field based on its label
+	 *
+	 * @return void
+	 */
+	public function selectOptionInRadioField($label, $option)
+	{
+		$I = $this;
+		$I->comment("Trying to select the $option from the $label");
+		$label = $webDriver->findField(['xpath' => "//label[contains(normalize-space(string(.)), '" . $label . "')]"]);
```

Comment:
Does `$I->click('Yes', '#jform_allowUserRegistration')`; not work?

<img width="778" alt="screenshot 2016-06-28 19 50 24" src="https://cloud.githubusercontent.com/assets/897638/16426499/9406d0c4-3d69-11e6-99c9-6ba06f942532.png">